### DISCUSSION TITLE
Remove shadow on a customized window

### DISCUSSION THREAD

**Discussion by DanKyungu** [AvaloniaUI/Avalonia#6817] [Q&A]
I need to have a custom window with rounded borders, I've applied properties mentioned below on the window and put a Border in the window as parent control with a CornderRadius and a padding but the final UI I got a window with transparent squared corners.

I think the problem is that, the window still have shadow on it and if shadow is removed from the window, the problem should be fixed.

It happening on window (Avalonia 0.10.8)

Properties applied on the windows :
CanResize="True" WindowStartupLocation="CenterScreen" ExtendClientAreaToDecorationsHint="True" ExtendClientAreaChromeHints="NoChrome" ExtendClientAreaTitleBarHeightHint="-1" TransparencyLevelHint="Transparent" SystemDecorations="None" Background="Transparent" UseLayoutRounding="True"

Result :
![image](https://user-images.githubusercontent.com/33684517/138554548-e9215163-7ab8-4f0e-b27d-91c744bfdcee.png)

**Comment by yegorpetrov** [comment]
Seriously, how to remove the shadow and the border in Windows 11? I tried setting a bunch of properties to make the window transparent, and it does become transparent, but the border and the shadow are there anyway.

![image](https://github.com/user-attachments/assets/8d28cd3d-6d53-428b-9cb3-843ae876ba93)

**Comment by PurelyAndy** [comment]
Removing the line ExtendClientAreaToDecorationsHint="True" fixed the issue for me

**Reply by yegorpetrov** [reply]
> Removing the line ExtendClientAreaToDecorationsHint="True" fixed the issue for me

Huh. It really does help.

![image](https://github.com/user-attachments/assets/bc86220d-025c-428b-a29a-a8c6865b1c51)
